### DISCUSSION TITLE
support Long Video demo   支持 Long Video demo

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#279] [MMAction2]
# support Long Video demo  
  支持 Long Video demo  

Support long video inference demo to predict the action categories of long video inputs at different times:
-You can refer to the demo/long in the 0. x branch_ video_ demo.py"

支持 长视频推理 demo，预测长视频输入在不同时刻的动作类别：
- 可以参考 0.x 分支中的demo/long_video_demo.py

## Task Information

**Repository**：MMAction2

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：None

**Claim Time**：None

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by jts250** [comment]
Claim the mission

**Reply by crazysteeaam** [reply]
Dear contributor, this task has been done by another contributor through this PR https://github.com/open-mmlab/mmaction2/pull/274, please choose another task with the label `StageAwaitAssignment`, thanks.

**Reply by jts250** [reply]
OK, thanks!
